sklearn/cluster/dbscan_.py

```diff
@@ -24,7 +24,7 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski',
     Parameters
     ----------
     X : array or sparse (CSR) matrix of shape (n_samples, n_features), or \
-            array of shape (n_samples, n_samples)
+array of shape (n_samples, n_samples)
```

Comment:
I frankly dislike this outdenting in the source code, which is at least as important as `help` output. I'd rather investigate whether numpydoc has a mechanism for handling oversize parameter type descriptions, and if not, proposing one at the numpydoc project (and again, working towards merging the divergent implementation in scikit-learn).